nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -80,6 +85,37 @@ in {
 
   options.services.nextcloud = {
     enable = mkEnableOption (lib.mdDoc "nextcloud");
+
+    enableBrokenCiphersForSSE = mkOption {
+      type = types.bool;
+      # Workaround can be removed at backport-time for 22.11.
+      default = !(versionOlder stateVersion "22.11");
```

Comment:
Please add `defaultText` here: otherwise building the manual depends on actual system config (even though it's not too much of an issue since stateVersion rarely changes).